openhands/runtime/client/runtime.py

```diff
@@ -115,7 +115,7 @@ def __init__(
         env_vars: dict[str, str] | None = None,
     ):
         self.config = config
-        self._port = find_available_tcp_port()
+        self._port = 30000
```

Comment:
```suggestion
        self._port = 30000 # initial dummy value
```

You mean it's never used, because it's computed downstream? Then if we need to init it at all, lets add a small comment